server/src/pages/shot/shotpage-header.js

```diff
@@ -31,7 +31,7 @@ exports.ShotPageHeader = class ShotPageHeader extends React.Component {
       );
     }
     return (
-      <Localized id="shotPageAllShotsButton">
+      <Localized id="shotPageAllShotsButton" attrs={{title: true}}>
         <a className="nav-button icon-shots" href="/shots" tabIndex="0" title="All Shots">
           <Localized id="shotPageAllShots">
             <span>All Shots</span>
```

Comment:
Missed one spot here: see `expirationSubtitle` in the `renderShotInfo()` method at line 52 of this file. Currently, instead of rendering the shot expiration component, I just see the string 'timediff':

![image](https://user-images.githubusercontent.com/96396/45902407-2ef21e00-bd9b-11e8-8fc0-c1d0ffac8fb7.png)

Should look something like this:

![image](https://user-images.githubusercontent.com/96396/45902483-6e206f00-bd9b-11e8-9d7a-9463584538ed.png)

Reply:
Thanks for catching it. I was able to run the addon locally which allowed me to fix this.